code/index.html

```diff
@@ -3,28 +3,83 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
+    <title>The Tea Nook</title>
+    <link rel="stylesheet" href="style.css">  
   </head>
+
   <body>
-    <!-- video or image as a header is cool :) -->
+    <header class="video-header">
+        <h1>The Tea Nook</h1>
+        <div class="video-container">
+          <div class="color-overlay">
+            <video autoplay loop muted>
```

Comment:
If you want to autoplay the video a tip is to add `playsinline` aswell, because some browsers in phones won't autoplay otherwise. You can read about it [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/video)